assets/src/blocks/Articles/ArticlesList.js

```diff
@@ -7,7 +7,7 @@ export const ArticlesList = (props) => {
     <div className="article-list-section clearfix">
       {posts && posts.length > 0 && posts.map(post =>
         <ArticlePreview
-          key={post.post_title}
+          key={post.ID}
```

Comment:
Post titles are not guaranteed to be unique. While developing you often end up with an Articles block on each page with at least 2 times "Test" as the title.